### DISCUSSION TITLE
Throwback version

### DISCUSSION THREAD

**Discussion by miloumeeloo** [levito/tt-rss-feedly-theme#109] [Q&A] [answered]
Hi,
Is there a way to use your theme with 19.2 (734af33) claro version?

**Comment by levito** [comment, marked as answer]
Hi, just click the Releases tab, every release states which TT-RSS version it's compatible with. For 19.2, please try https://github.com/levito/tt-rss-feedly-theme/releases/tag/v2.3.1

Edit: The theme is not compatible with the [claro-throwback](https://git.tt-rss.org/fox/tt-rss/commits/branch/claro-throwback) branch.

**Reply by miloumeeloo** [reply]
Thanks, i tried all version from 2.1.0 to 2.3.1 and always getting this with feed tree, feed editor, menus (including right-click) and config page.
![feedly1](https://user-images.githubusercontent.com/57871403/152664683-6bca3034-8fd7-410c-b863-e6c8f35ca6ea.png)
![feedly2](https://user-images.githubusercontent.com/57871403/152664688-0ee92c49-4926-415a-8adf-93960d85bca5.png)
![feedly3](https://user-images.githubusercontent.com/57871403/152664695-30a1631d-7699-484f-96bb-0f985b6d51bc.png)
![feedly4](https://user-images.githubusercontent.com/57871403/152664697-3d7d898d-851f-49da-9b8e-7570e9d99ebc.png)

**Reply by levito** [reply]
This looks strange... I just tried your mentioned tt-rss commit 734af33 and theme version 2.3.1 – it works. Which OS and which browser in which version are you using? I checked with Chrome 97, Safari 15.3 and Firefox 95 on macOS Big Sur.

**Reply by miloumeeloo** [reply]
I'm on Ubuntu 20.04 with Firefox 96.  I use ttrss from the claro-throwback branch (non-dockerized)

**Reply by levito** [reply]
Oh, now I see the problem: You are not using the commit [734af33](https://git.tt-rss.org/fox/tt-rss/src/commit/734af33) which I checked, but that old [claro-throwback](https://git.tt-rss.org/fox/tt-rss/commits/branch/claro-throwback) branch. The theme is not compatible with it. It uses a different dojo theme and therefore has completely different HTML. That's why especially UI elements like buttons, inputs, modals etc. are totally broken.

Why do you use that branch? I'd strongly suggest to switch to master, also to get potential security fixes. The version that you use is already 3 years old.

**Reply by miloumeeloo** [reply]
Ok, sorry, I got confused between the different branches.
I'm now using the master branch and everything works fine. Thanks a lot for your support.
